pintc/src/error/compile_error.rs

```diff
@@ -383,6 +383,8 @@ pub enum CompileError {
     },
     #[error("invalid map range type")]
     InvalidMapRangeType { found_ty: String, span: Span },
+    #[error("invalid constant")]
+    InvalidConst { span: Span },
```

Comment:
Maybe something closer to Rust:
![image](https://github.com/user-attachments/assets/84250ec7-0b39-4989-ae16-af256c42fe0a)